### PR TITLE
Added iOS8 preload-top-hits issue fix for v7 tag

### PR DESCRIPTION
**Pull request by fractaltheory**
Status: **Ready for Review**
Owner: @donnielrt
Reviewers: @haroldtreen @jansepar 
## Jira Tickets:
- [RTM-281](https://mobify.atlassian.net/browse/RTM-281)
## Todos:
- [x] Don +1
- [x] Shawn +1
- [x] Minified version
### Feedback:

_none so far_
## Changes
- Add a workaround in the v7 tag for an iOS8 issue, where the autocomplete request for a site sets a mobify-path cookie to blank
- Use the page visibility API to detect if we're in a background tab, in which case we don't drop an opt-out cookie
## How to Test
- Use Charles to rewrite to this tag, and verify that the `mobify-path` cookie isn't being set

### CODE REVIEW

**Comment by donnielrt** [comment]
Thanks for getting this started @fractaltheory, taking a look now.

File: tag/v7.js [hunk 1]
```diff
@@ -202,6 +202,21 @@ Private['loadPreview'] = loadPreview;
...
+    // background when typing in an address into the URL bar. Safari
+    // seems to cancel the mobify.js download, resulting in us dropping
+    // a failure cookie.
+    //
+    // We get around this by ignoring mobify.js failures in a background
+    // tab.
+    //
+    // Related ticket: https://mobify.atlassian.net/browse/RTM-280
+    //
+    if (doc.visibilityState && doc.hidden) {
```
Comment (donnielrt) [inline tag/v7.js]: There's no "doc" here :) We need to change this to document

Comment (fractaltheory) [inline tag/v7.js]: Oh geez. Fixed: 269cb6f7255f33d506dab0429bedd5676ef734fd

**Comment by donnielrt** [comment]
Confirmed that this fixes the issue for the v7 tag as well, woohoo!

:+1:

**Comment by donnielrt** [comment]
@jansepar carrying on the discussion from the [v6 tag PR](https://github.com/mobify/mobify-client/pull/78#discussion_r35720225), do you think we need to make this fix opt-out rather than opt-in? I vote yes.

**Comment by donnielrt** [comment]
@jansepar can I get a hallelujah?

**Comment by jansepar** [comment]
Looks pretty legit! :+1:

**Comment by donnielrt** [comment]
Thanks Shawn!

**Pull request merged**